lib/search-bar/defaultSuggestions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * If there is no suggestions matches this list will be displayed
+ */
+export default [{
+  boldTitle: 'Oh no! ',
```

Comment:
I would go with "Desværre ingen resultater på din søgning..."
and "Måske kan du prøve følgende i stedet"
![errormessage](https://cloud.githubusercontent.com/assets/8721221/16787939/af5aed3c-48a0-11e6-9ed2-c96216c38da5.png)